editor/store/test/effects.js

```diff
@@ -471,6 +474,8 @@ describe( 'effects', () => {
 
 	describe( 'reusable block effects', () => {
 		beforeAll( () => {
+			lodash.uniqueId = jest.spyOn( lodash, 'uniqueId' );
```

Comment:
I wonder if we could achieve this more concisely with a combination of `jest.mock` and `require.requireActual`.

But then, looking at how you're asserting on `optimist.id`, do we really need this at all? Since you're using ` expect.any( Object ),` and not asserting against the actual value produced by `uniqueId`.

Reply:
You're right, mocking this isn't giving us much benefit for the effort involved. I've removed the mock in 78a9bb7657be82cc037d68ca121b27b6c68f75f7.